schemas/json/layout/expression.schema.v1.json

```diff
@@ -463,6 +518,29 @@
       ],
       "additionalItems": false
     },
+    "func-stringSlice": {
+      "title": "String slice function",
+      "description": "This function returns a substring of a string",
+      "type": "array",
+      "items": [
+        { "const":  "stringSlice" },
+        { "$ref": "#/definitions/string" },
+        { "$ref": "#/definitions/number" },
+        { "$ref": "#/definitions/number" }
+      ],
+      "additionalItems": false
+    },
+    "func-stringIndexOf": {
+      "title": "String index of function",
+      "description": "This function returns the index of the first occurrence of a substring in a string",
```

Comment:
What happens if the substring does not occur? That should probably be mentioned

Reply:
It's a bitt iffy, because now it returns `null` (not `-1`), and `stringSlice` breaks with a runtime error when you pass `null` to it. That's intentional, of course, and it especially useful when I add support for negative values in `stringSlice`, as that would just behave strangely if used together with `stringIndexOf` when it returns `-1`.